openapi/paths/quotes@{id}@recall.yaml

```
@@ -0,0 +1,26 @@
+parameters:
+  - $ref: ../components/parameters/resourceId.yaml
+post:
+  x-products:
+    - Core
+  tags:
+    - Quotes
+  summary: Recall a quote
+  operationId: PostQuoteRecall
+  x-sdk-operation-name: recall
+  description: Recalls an issued quote with a specified ID. After recall the quote status is `draft`.
+  responses:
+    '200':
+      description: Quote recalled.
```

nitpick: Related to above comment
```suggestion
      description: Quote retracted.
```